src/commands/ssh-key/rm.ts

```diff
@@ -21,7 +21,7 @@ import * as cf from '../../utils/common-flags';
 import { getBalenaSdk, stripIndent } from '../../utils/lazy';
 import { parseAsInteger } from '../../utils/validation';
 
-export default class KeyRmCmd extends Command {
+export default class SSHKeyRmCmd extends Command {
```

Comment:
No aliases & deprecateAliases in this command?

Reply:
Nice catch